ARCs/arc-0033.md

```diff
@@ -0,0 +1,60 @@
+---
+arc: 33
+title: Becoming an xGov
+description: Explanation on how to become Expert Governors.
+author: Stéphane Barroso (@SudoWeezy), Adriana Belotti, Massimo Morini, Michel Treccani, John Woods, Shai Halevi
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/
+status: Draft
+type: Meta
+created: 2022-11-22
+---
+ 
+## Abstract
+xGov status gives a governor the right to vote on proposals raised by the community, to ratify the proposals as valid for final community vote, where said proposals pertain to spending a previously specified amount of Algo in a given epoch on particular initiatives.
```

Comment:
In general, I'm curious what part the Foundation and/or Algorand Ventures, will have in the day-to-day operations, support, and implementation of xGov ratified proposals and the ratification process in general as well as any potential internal or utility measures for the xGov program itself.

Reply:
Unless I am mistaken, the intent here is:
1. For the community to be able to write proposals on which xGovs can vote, and
2. Proposals, if promoted by xGovs to a full governance measure and approved by a majority of all governors, would always be implemented by the Foundation.  

In neither case--proposal writing or approved measure implementation--would this involve Algorand Ventures.  Someone please correct me if I am under the wrong impression.